### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/phoenix/commit/7987739578809a8a1956b484101ec7aaa6b2ef66#commitcomment-80372549

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/phoenix/commit/d7b86f247075ca91e7e119e96233a9f50607b93d

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: guides/asset_management.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ $ yarn add --dev esbuild
 $ yarn add ../deps/phoenix ../deps/phoenix_html ../deps/phoenix_live_view
 ```
 
-Next, add a custom Javascript build script. We'll call the example `assets/build.js`:
+Next, add a custom JavaScript build script. We'll call the example `assets/build.js`:
```
Comment (jsoref) [inline guides/asset_management.md:87]: brand

File: test/phoenix/verified_routes_test.exs [hunk 2]
```diff
@@ -176,7 +176,7 @@ defmodule Phoenix.VerifiedRoutesTest do
     end
   end
 
-  test "~p raises on dynamic dynamic interpolation" do
+  test "~p raises on dynamic interpolation" do
```
Comment (jsoref) [inline test/phoenix/verified_routes_test.exs:179]: I'm not sure about this

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine.

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**